frontend/js/jsonify.js

```diff
@@ -0,0 +1,97 @@
+//Todo
+function jsonifyCharacter(){
+
+	var htmlCharFields = document.getElementsByClassName("add-character-value");
+
+	 for(var i = 0;i < htmlCharFields.length;i++){//why does for var field in htmlcharfields not work?
```

Comment:
the following should work and do exactly what you want. The syntax is a bit cleaner

for( var fieldIndex in htmlCharFields )
{
    htmlCharFields[fieldIndex] ...
}